datafusion/sqllogictest/test_files/tpch/q11.slt.part

```diff
@@ -75,10 +75,10 @@ Limit: skip=0, fetch=10
 ----------------------TableScan: nation projection=[n_nationkey, n_name], partial_filters=[nation.n_name = Utf8("GERMANY")]
 physical_plan
 GlobalLimitExec: skip=0, fetch=10
---SortPreservingMergeExec: [value@1 DESC], fetch=10
-----SortExec: TopK(fetch=10), expr=[value@1 DESC]
-------ProjectionExec: expr=[ps_partkey@0 as ps_partkey, SUM(partsupp.ps_supplycost * partsupp.ps_availqty)@1 as value]
---------NestedLoopJoinExec: join_type=Inner, filter=CAST(SUM(partsupp.ps_supplycost * partsupp.ps_availqty)@0 AS Decimal128(38, 15)) > SUM(partsupp.ps_supplycost * partsupp.ps_availqty) * Float64(0.0001)@1
+--SortExec: TopK(fetch=10), expr=[value@1 DESC]
+----ProjectionExec: expr=[ps_partkey@0 as ps_partkey, SUM(partsupp.ps_supplycost * partsupp.ps_availqty)@1 as value]
+------NestedLoopJoinExec: join_type=Inner, filter=CAST(SUM(partsupp.ps_supplycost * partsupp.ps_availqty)@0 AS Decimal128(38, 15)) > SUM(partsupp.ps_supplycost * partsupp.ps_availqty) * Float64(0.0001)@1
+--------CoalescePartitionsExec
```

Comment:
This does seem to use run less in parallel?

Reply:
Yes, inner (always left) input must be collected into single partition, and outer (always right) input is executed in parallel. In cases reordering is not happening due to lacking / misestimated  statistics -- it'll also cause lack of parallelism.

Reply:
> In case reordering is not happening due to lacking / misestimated statistics -- it'll also cause lack of parallelism.

`should_swap_join_order` is based on `total_byte_size`.  It would become `Precision::Absent` after some operator like `AggregateExec`.  I don't know whether it is a good idea to add a check by `num_rows`.

Reply:
It already falls back to num_rows if any of join inputs doesn't provide bytes statistics.